phoenix5-spark/src/it/resources/globalSetup.sql

```diff
@@ -60,7 +60,7 @@ UPSERT INTO "small" VALUES ('key1', 'foo', 10000)
 UPSERT INTO "small" VALUES ('key2', 'bar', 20000)
 UPSERT INTO "small" VALUES ('key3', 'xyz', 30000)
 
-CREATE TABLE MULTITENANT_TEST_TABLE (TENANT_ID VARCHAR NOT NULL, ORGANIZATION_ID VARCHAR, GLOBAL_COL1 VARCHAR  CONSTRAINT pk PRIMARY KEY (TENANT_ID, ORGANIZATION_ID)) MULTI_TENANT=true
+CREATE TABLE MULTITENANT_TEST_TABLE (TENANT_ID VARCHAR NOT NULL, ORGANIZATION_ID VARCHAR, GLOBAL_COL1 VARCHAR  CONSTRAINT pk PRIMARY KEY (TENANT_ID, ORGANIZATION_ID)) MULTI_TENANT=true, SALT_BUCKETS = 20
```

Comment:
If we want to test salting, we should create a new table and new tests for that.

Reply:
I added a new multitenant table with salt to test multitenant and salted case.